src/ccl_tracers.c

```diff
@@ -385,11 +388,15 @@ void ccl_get_kappa_kernel(ccl_cosmology *cosmo, double chi_source,
   for (int ichi=0; ichi < nchi; ichi++) {
     double chi = chi_arr[ichi];
     double a = ccl_scale_factor_of_chi(cosmo, chi, status);
-    double mgfac = 1;
+ //   double mgfac = 1;
```

Comment:
same thing here, don't forget to delete commented out lines.

Reply:
removed too!